src/Basque/Mews.Fiscalizations.Basque/Mews.Fiscalizations.Basque.csproj

```diff
@@ -10,7 +10,7 @@
     <RepositoryUrl>https://github.com/MewsSystems/fiscalizations</RepositoryUrl>
     <Icon>https://raw.githubusercontent.com/msigut/eet/master/receipt.png</Icon>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <PackageVersion>2.0.3</PackageVersion>
+    <PackageVersion>2.0.4</PackageVersion>
```

Comment:
Following the [semantic versioning](https://semver.org/), shouldn't we bump the minor version on changes like this?